src/orders/orders.cpp

```diff
@@ -20,10 +20,101 @@
 
 namespace antara::orders
 {
+    order::order(st_order_id &id, antara::pair &pair, st_price &price,
+                st_quantity &quantity, st_quantity &filled, antara::side &side,
+                order_status &status)
```

Comment:
This need to be initialized with a braces list initialization https://en.cppreference.com/w/cpp/language/initializer_list